stdlib/trace.pyi

```diff
@@ -41,7 +43,10 @@ class Trace:
         globals: Optional[Mapping[str, Any]] = ...,
         locals: Optional[Mapping[str, Any]] = ...,
     ) -> None: ...
-    def runfunc(self, func: Callable[..., _T], *args: Any, **kw: Any) -> _T: ...
+    if sys.version_info >= (3, 9):
+        def runfunc(self, __func: Callable[..., _T], *args: Any, **kw: Any) -> _T: ...
+    else:
+        def runfunc(self, func: Callable[..., _T], *args: Any, **kw: Any) -> _T: ...
```

Comment:
This might be a good place to use `ParamSpec`.

Reply:
would this result in 

```python
_P = ParamSpec("_P")

def runfunc(self, __func: Callable[..., _P], *args: Any, **kw: Any) -> _P: ...
```

Never used it before, but after reading about it, it makes sense. Thanks for the pointer 🙂

Reply:
You can use a `TypeVar` for the  type, just like you did. `ParamSpec` is for parameters. If I have understood it correctly:

```python
_T = TypeVar("_T")
_P = ParamSpec("_P")

def runfunc(self, __func: Callable[_P, _T], *args: _P.args, **kwargs: _P.kwargs) -> _T: ...
```